lapack/testlapack/dlatrs.go

```diff
@@ -103,8 +103,11 @@ func dlatrsResidual(uplo blas.Uplo, trans blas.Transpose, diag blas.Diag, n int,
 		}
 	}
 
-	eps := dlamchE
-	smlnum := dlamchS
+	const (
+		eps    = dlamchE
+		smlnum = safmin
```

Comment:
This name collision is unfortunate.

Reply:
Is it worth calling this something else, `small`, to differentiate it?

Reply:
Yes, that's better. I went with `tiny` and `huge` which is quite accurate and differentiates them from smlnum and bignum.